anet.yml

```diff
@@ -330,3 +330,9 @@ dictionary:
         options:
           layers: GEBCO_LATEST
           format: "image/png"
+
+  automaticallyInactivateUsers:
+    emailReminders:
+      daysPrior: [15, 30, 45]
+    clearDomainUserName:
+      test: []
```

Comment:
The property is called `clearDomainUserName.test` but from inspecting the code it appears to be for a list of domains from which users should not be inactivated. Please give the entry a more descriptive name (and probably the just collapse the sub-key since there will be no other properties next to it); something like `ignoredDomainNames` would be clearer.

Reply:
Agreed, but we could also do this for emailsReminders.daysPrior by refactoring it to emailRemindersDaysPrior: []